src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Engine/Trivia/CSharpTriviaFormatter.cs

```diff
@@ -354,5 +354,20 @@ private SyntaxTrivia FormatDocumentComment(LineColumn lineColumn, SyntaxTrivia t
 
             return GetLineColumnDelta(lineColumn, docComment);
         }
+
+        protected override bool IsLineContinuation(SyntaxTrivia trivia)
+        {
+            return false;
+        }
+
+        /// <summary>
+        /// This is only here to make interface happy
+        /// </summary>
+        /// <param name="trivia"></param>
+        /// <returns>True is trivial is a comment</returns>
+        protected override bool IsComment(SyntaxTrivia trivia)
+        {
+            return trivia.IsDocComment();
+        }
```

Comment:
Why this looks specifically for doc comments?

Reply:
agreed.  this is super strange.  it is called IsComment, but only returns true if the trivia is a DocComment.

Reply:
It will never be called and I didn't know how it should be handled in case in the future someone wants to use it.

Reply:
Fixed and renamed, it now throws in C# and is renamed to IsVIsualBasicComment